src/components/AppMain/NewCard/NewCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { CardContext } from '../../Context/CardListProvider'
+import styles from './NewCard.module.css'
+import React, { ChangeEvent, useContext } from 'react'
+
+const NewCard = () => {
+	const { setEditingCard, editingCard, setIsAdding, isSecondSide, setIsSecondSide, setCardList } =
```

Comment:
**nit:** setting card side should be a part of local state - not provider state.